Holovibes/mainwindow.ui

```diff
@@ -1152,6 +1185,9 @@
            </item>
            <item row="3" column="2">
             <widget class="QLabel" name="QaccLabel">
+             <property name="toolTip">
+              <string>Interval to include after 'z2' (max #img)</string>
```

Comment:
Number of frequencies kept by the  SVD/SSA algorithm : the range of frequencies kept is [z2, z2 + width].